### DISCUSSION TITLE
How can I have a global filter with data including array?

### DISCUSSION THREAD

**Discussion by fpicoitoj** [TanStack/table#5085] [Q&A] [answered]
Hello everyone,

This might be a very noob question but I am quite stuck.

I cannot make the global filter also search for string[] data, as it seems to completely ignore it. I also cannot understand how the column`s filterFn function is supposed to work, as my expectation was that it would make a given field behave a certain way.

I`ve changed the code I share now in multiple ways (on my project), including not making a fuzzy function, and just use a global filter, but what appears to happen is that the column with the string[] values is not called by the global filter function.

https://codesandbox.io/p/sandbox/shy-water-xy3cjz

I appreciate any insight and I apologize in advance if somehow this is completely missing me and I would gladly have you point me towards the necessary documentation and\or solution.

Thank you very much.

Update:
If you have the same issue, one workaround, should your case be as simple as a string[], just make a new column referencing a string, and join your array comma separated, and the function will search that data.

**Comment by fpicoitoj** [comment, marked as answer]
To anyone who finds this:

https://tanstack.com/table/v8/docs/api/features/filters#can-filter

What is missing in the code is adding the following snippet in useReactTable
```
getColumnCanGlobalFilter: (column) => {
     // any code you need
    return true; 
},
```

By doing this, the column will also be caught in the global filter.

**Reply by Eiley2** [reply]
Man, you just made my life better almost 2 years later.
